lesson6/src/test/java/com/andersenlab/arrayList/CustomArrayListTest.java

```diff
@@ -0,0 +1,79 @@
+package com.andersenlab.arrayList;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Iterator;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid generic imports with *. You could configure imports in intellijIdea so it will always use full imports